packages/shared/src/logger.ts

```diff
@@ -93,6 +94,9 @@ class Logger {
   }
 
   private log(message: string, level: LogLevel, tags?: Tags) {
+    if (!this.initialized) {
+      throw new Error("Logger not initialized. Call `logger.initialize` first.");
+    }
```

Comment:
This won't play nicely with the Playwright fixture. We can't quite initialize the logger there because we have no place to close it in that context (unless we'd be initializing and closing the logger for each test separately). So I'm torn on what should be done about it.

Reply:
Should we do what we do elsewhere and queue such logs up until the logger has been initialized?

Reply:
We don't have a good communication channel between the reporter and the reporter (which has access to the logger). Currently, we use WS to communicate them... but I'm just literally working on replacing that with Playwright's annotations/attachments. I don't think we should be keeping our logs there though.

Reply:
> We don't have a good communication channel between the reporter and the reporter

🤔 Huh? 😅 lol

Reply:
Reporter and fixture?

Reply:
I see.

So this change means...if anything in the fixture tries to log, it will throw? Does it throw already? (is there already logging in that context?)

Reply:
> So this change means...if anything in the fixture tries to log, it will throw?

Yes, for that reason it's probably better to revert this part of the PR.

> Does it throw already?

No

> is there already logging in that context?

Just local debug-based one under the `replay` namespace (the default one). We don't have grafana logs in this context

Reply:
Ok 👍🏼